assets/src/dashboard/app/views/templates/detail.js

```diff
@@ -245,7 +258,9 @@ function TemplateDetail() {
                     <UnitsProvider pageSize={pageSize}>
                       <StoryGridView
                         stories={relatedTemplates}
-                        centerActionLabel={__('View', 'web-stories')}
+                        centerActionLabelByStatus={
+                          TEMPLATES_GALLERY_ITEM_CENTER_ACTION_LABELS
```

Comment:
One nit that you don't need to change now but should be mindful of is that I'm pretty sure, once we actually start pulling templates from the db, templates will just have a different post type or something and _not_ a status of "template".  That's our temporary way to separate them until we actually get them rigged up via the API, so the `ByStatus` of the `centerActionLabelByStatus` may not be the long term determinant of what the center label may be.

Reply:
that's kinda what i expected! This way we can have something to match whatever we want the 'status' to be (or in the case of templates whatever type of thing it is). I expect we'll have some massaging to do of the story grid items to play nicely but this was one step towards having things be done logically instead of hard coding strings.

Reply:
I can dig it 👍